src/client/src/orangehrmAdminPlugin/pages/languagePackage/languagePackage.vue

```diff
@@ -0,0 +1,188 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template lang="">
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-paper-container">
+      <div class="orangehrm-header-container">
+        <oxd-text tag="h6" class="orangehrm-main-title">
+          {{ $t('general.language_packages') }}
+        </oxd-text>
+        <div>
+          <oxd-button
+            :label="$t('general.add')"
+            icon-name="plus"
+            display-type="secondary"
+            @click="onClickAddLanguage"
+          />
+        </div>
+      </div>
+      <table-header :total="total"></table-header>
+      <div class="orangehrm-container">
+        <oxd-card-table
+          v-model:selected="checkedItems"
+          v-model:order="sortDefinition"
+          :headers="headers"
+          :items="items?.data"
+          :selectable="false"
+          :clickable="false"
+          :loading="isLoading"
+          row-decorator="oxd-table-decorator-card"
+        />
+      </div>
+      <div class="orangehrm-bottom-container">
+        <oxd-pagination
+          v-if="showPaginator"
+          v-model:current="currentPage"
+          :length="pages"
+        />
+      </div>
+    </div>
+    <br />
+    <add-language-modal
```

Comment:
add-language-modal will open outside of the normal dom no need for `<br/>`

Reply:
Done